vlmeval/vlm/mplug_owl3.py

```diff
@@ -0,0 +1,135 @@
+import torch
+from .base import BaseModel
+from ..smp import *
+from ..dataset import DATASET_TYPE
+from transformers import AutoTokenizer, AutoModel
+
+
+class mPLUG_Owl3(BaseModel):
+    # No separate model module is required, but the dependencies must be met.
+    # https://github.com/X-PLUG/mPLUG-Owl/blob/main/mPLUG-Owl3/requirements.txt
+    INSTALL_REQ = True
+    INTERLEAVE = True
+    INSTALL_REQ_TXT = 'https://github.com/X-PLUG/mPLUG-Owl/blob/main/mPLUG-Owl3/requirements.txt'
+
+    def __init__(self, model_path=None, **kwargs):
+        assert model_path is not None
+        self.tokenizer = AutoTokenizer.from_pretrained(
+            model_path,
+            trust_remote_code=True
+        )
+        from mplug_owl3_dsy import mPLUGOwl3Model
```

Comment:
Do not include custom names in the PR.
You can refer to the implementation of mPLUG-Owl2 and assume that mPLUG-Owl3 has been installed properly.

Reply:
Sorry, here i was trying to test the difference between loading with automodel and the way it's loaded in the official demo, I'll fix this.